packages/sui-studio/src/components/style/index.js

```diff
@@ -19,8 +19,12 @@ export default class Style extends Component {
     this._linkElement = createLinkElement()
   }
 
+  componentWillUnmount () {
+    const lastIndex = document.styleSheets.length - 1
+    document.styleSheets[lastIndex].disabled = true
+  }
+
   render () {
-    // https://github.com/webpack-contrib/style-loader/blob/master/addStyles.js#L238
```

Comment:
The new: https://github.com/webpack-contrib/style-loader/blob/master/lib/addStyles.js